### ISSUE TITLE
server/util-server.js: ping: missing error messages from stderr

### ISSUE THREAD

**Issue by wonderfulShrineMaidenOfParadise** [louislam/uptime-kuma#2896]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

![Screenshot 2023-03-08 at 07-09-30 Uptime Kuma](https://user-images.githubusercontent.com/86928419/223645063-748a4ca9-b093-43fd-8527-4a9ec2a7faf0.png)
![Screenshot 2023-03-08 at 07-09-56 Uptime Kuma](https://user-images.githubusercontent.com/86928419/223645070-506f343a-1ca8-4912-9cd0-d52b88199951.png)

### 👟 Reproduction steps

Add a ping monitor that returns errors.

### 👀 Expected behavior

Messages from stderr are displayed in web ui / recorded in logs so that users can do troubleshooting next.

### 😓 Actual Behavior

The responses don't have messages from stderr included.

### 🐻 Uptime-Kuma Version

1.20.2

### 💻 Operating System and Arch

Alpine 3.17 armv7

### 🌐 Browser

-

### 🟩 NodeJS Version

18.14.2

### 📝 Relevant log output

```shell
2023-03-08T07:09:50Z [MONITOR] WARN: Monitor #30 '1.1.1.1': Failing: PING 1.1.1.1 (1.1.1.1): 56 data bytes
 | Interval: 60 seconds | Type: ping | Down Count: 0 | Resend Interval: 0
2023-03-08T07:09:58Z [MONITOR] WARN: Monitor #29 ' p.p': Failing:  | Interval: 60 seconds | Type: ping | Down Count: 0 | Resend Interval: 0
```
```

**Comment by louislam** [comment]
You should ask on https://github.com/danielzzz/node-ping

**Comment by louislam** [comment]
For packet loss error, it does report error. That is why I think it is an issue from the upstream library.

![image](https://user-images.githubusercontent.com/1336778/223650568-c9b237b7-dcc6-400f-ab3e-a52753a6ffa3.png)

Since I can't reproduce it on my system, if you know some node.js, you could confirm it by utilizing node-ping directly. 

Example from `node-ping`
```
var ping = require('ping');

var hosts = ['192.168.1.1', 'google.com', 'yahoo.com'];

hosts.forEach(function (host) {
    ping.promise.probe(host)
        .then(function (res) {
            console.log(res);
        });
});
```

**Comment by wonderfulShrineMaidenOfParadise** [comment]
![Screenshot 2023-03-11 at 22-13-38 Uptime Kuma](https://user-images.githubusercontent.com/86928419/224513762-4c72292d-9709-4907-8032-d7556f336c2e.png)
![Screenshot 2023-03-11 at 22-13-48 Uptime Kuma](https://user-images.githubusercontent.com/86928419/224513767-38eb64fa-990e-431c-8646-0b48ab1f446b.png)

It has been fixed in upstream, please bump node-ping version to 0.4.4.

**Comment by wonderfulShrineMaidenOfParadise** [comment]
Fixed by 1e80365b739a83192c55701fa2f30454666e9907.
